buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -164,6 +183,9 @@
     },
     "GITHUB_PASSWORD": {
       "value": "PassedViaPipeBuild"
+    },
+    "PB_DockerImageName": {
```

Comment:
>PB_DockerImageName [](start = 5, length = 18)

Is it strange and deceiving to name this with the PB_ prefix when the variable is really local and not specified by the pipe build?